tools/public_api_guard/material/tooltip.md

```diff
@@ -107,10 +107,19 @@ export abstract class _MatTooltipBase<T extends _TooltipComponentBase> implement
     _overlayRef: OverlayRef | null;
     get position(): TooltipPosition;
     set position(value: TooltipPosition);
-    show(delay?: number): void;
+    // (undocumented)
+    get positionAtOrigin(): boolean;
+    set positionAtOrigin(value: BooleanInput);
+    show(delay?: number, origin?: {
+        x: number;
+        y: number;
+    }): void;
     get showDelay(): number;
     set showDelay(value: NumberInput);
-    toggle(): void;
+    toggle(origin?: {
```

Comment:
@devversion this is just now showing up, does this mean that our api guard wasn't properly finding the parameters before?

Reply:
I think the API file just hasn't been approved/updated in `main` (hence the job being red there)